docs/btc-methods.md

```diff
@@ -0,0 +1,73 @@
+# BTC Methods
+
+Here we document the Bitcoin methods that an account Snap may implement to
+support requests originated from dapps.
+
+## btc_sendmany
+
+This method is similar to the `sendmany` RPC method from Bitcoin Core, but its
+parameters are passed in an object instead of an array, and are named in
+camelCase. Also, dummy values aren't allowed.
+
+### Parameters
+
+- **Request object (required):**
+  - Type: `object`
+  - Properties:
+    - `amounts`
+      - Description: A JSON object with recipient addresses and amounts.
+      - Type: `object`
+      - Properties:
+        - `[key]: string`: Address of the recipient
+        - `[value]: string`: Amount to send to the recipient in BTC
+    - `comment` (optional)
+      - Description: A comment.
+      - Type: `string`
+    - `subtractFeeFrom` (optional)
+      - Description: The fee will be equally deducted from the amount of each
```

Comment:
_The addresses_ looks like a very bad description to me 😅 I'd prefer to be more descriptive here instead of being consistent with the original `sendmany` method.